frontend/src/components/AppHeader/AppHeader.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '../../test-utils';
```

Comment:
Zobacz czy zadzaiła :)
```suggestion
import { render } from 'test-utils';
```

Reply:
Niezadziałało?

Reply:
Działa :)